### PR TITLE
Creates attribute to store custom Get Tickets button label for events

### PR DESCRIPTION
**Pull request by clarissalimab**
Part of [#3381](https://github.com/CommitChange/tix/issues/3381).

### CODE REVIEW

File: db/migrate/20211104154340_add_get_tickets_button_label_to_events.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddGetTicketsButtonLabelToEvents < ActiveRecord::Migration
+  def change
+    add_column :events, :get_tickets_button_label, :string, default: nil
```
Comment (clarissalimab) [inline db/migrate/20211104154340_add_get_tickets_button_label_to_events.rb]: Do we want to also add the flag to `misc_event_info`?

Comment (wwahammy) [reply db/migrate/20211104154340_add_get_tickets_button_label_to_events.rb]: Yes, `misc_event_info` would be the right place.

**Review by wwahammy — CHANGES_REQUESTED** [review]
It all looks pretty solid but the migration should be on a separate PR so we can migrate the db and not risk any downtime.

**Review by wwahammy — APPROVED** [review]
:rocket:

**Pull request merged**